dns/bind/src/opnsense/mvc/app/controllers/OPNsense/Bind/forms/general.xml

```diff
@@ -33,6 +33,20 @@
         <type>text</type>
         <help>Set the port the service should listen to.</help>
     </field>
+    <field>
+        <id>general.transfersource</id>
+        <label>Transfer Source IP</label>
+        <type>text</type>
+        <advanced>true</advanced>
+        <help>This determines which local address is bound to IPv4 TCP connections used to fetch zones transferred inbound by the server.</help>
```

Comment:
Even technically correct, the help text is quite hard to understand.

```suggestion
        <help>Specify the IPv4 address used as a source for zone transfers.</help>
```

Reply:
It's the official description for this option, but I don't mind changing it.